starter-code/css/main.css

```diff
@@ -6,5 +6,61 @@ light blue: #1c5380
 
 body {
   font: 200 16px/1.5 "Helvetica Neue", Helvetica, Arial, sans-serif;
-  margin: 0;
+  border-top: 250px solid #125688;
+  margin: 20px 20px 20px 20px;
+}
+
+#phone {
+  position: relative;
+  float: left;
+  margin-top: -170px;
```

Comment:
Try avoiding using such large margin and padding values, especially when negative. This can wreak havok on the rest of the elements on your page and make the layout break when viewed outside of its optimal resolution.